packages/fhi-designsystem/src/components/fhi-tag/fhi-tag.stories.ts

```diff
@@ -0,0 +1,58 @@
+import type { Meta, StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import { FhiTag } from './fhi-tag.component';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { FhiIconRefresh } from '../icons/fhi-icon-refresh.component';
+import { FhiFlex } from '../fhi-flex/fhi-flex.component';
+import { FhiIconDownload } from '../icons/fhi-icon-download.component';
+import { FhiIconClock } from '../icons/fhi-icon-clock.component';
+
+new FhiTag();
+new FhiFlex();
+new FhiIconRefresh();
+new FhiIconDownload();
+new FhiIconClock();
+
+const meta: Meta<FhiTag> = {
+  title: 'Komponenter/Tag',
+  component: 'fhi-tag',
+  parameters: {},
+  decorators: [],
+  render: args => html`<fhi-tag color=${ifDefined(args.color)}>Tag</fhi-tag>`,
+  argTypes: {
+    color: {
+      options: ['neutral', 'accent', 'success', 'warning', 'danger', 'info'],
+      control: { type: 'select' },
+      description: 'Bestemmer fargetema.',
+      defaultValue: { summary: 'neutral' },
+    },
+  },
+};
+
+type Story = StoryObj<FhiTag>;
+
+export const Preview: Story = {
+  tags: [],
+  args: {},
+};
+
+export const Icon: Story = {
+  tags: ['!dev'],
+  render: () =>
+    html`<fhi-flex direction="row" wrap>
+      <fhi-tag color="neutral"
+        ><fhi-icon-download></fhi-icon-download>Fra Kristin</fhi-tag
```

Comment:
```suggestion
        ><fhi-icon-download></fhi-icon-download>Fra Folkeregisteret</fhi-tag
```

Cristin er et system det hentes data fra, men tenker dette er noe flere kjenner til.